src/shogun2-core/src/test/resources/META-INF/hibernate.properties

```diff
@@ -5,7 +5,7 @@ hibernate.id.new_generator_mappings=true
 hibernate.hbm2ddl.auto=create
 
 # caching
-#hibernate.cache.use_query_cache=true
+hibernate.cache.use_query_cache=false
```

Comment:
did you experience issues having set this to true? if so, which ones?

Reply:
Not really, but since this was implicitly the default before, I thought I'd leave it at that. I just need the property to be set, since I'm referencing it in the code.